frontend/public/components/replication-controller.jsx

```diff
@@ -60,7 +60,24 @@ const CancelAction = (kind, obj) => ({
   accessReview: asAccessReview(kind, obj, 'patch'),
 });
 
-const menuActions = [CancelAction, ...replicaSetMenuActions];
+const RollBackAction = (kind, obj) => ({
+  // t('public~Rollback')
+  labelKey: 'public~Rollback',
+  // disabled if the DC is not Active
+  isDisabled: INACTIVE_STATUSES.includes(
+    obj?.metadata?.annotations?.['openshift.io/deployment.phase'],
+  ),
+  // Hidden if RC is active or does not belong to a deployment config
+  hidden: obj?.status?.replicas > 0 || !getDeploymentConfigNameForRC(obj),
+  callback: () =>
+    rollBackRCModal({
+      resourceKind: kind,
+      resource: obj,
+    }),
+  accessReview: asAccessReview(kind, obj, 'patch'),
```

Comment:
Sorry, I missed this on last review, but the RBAC check looks wrong here. We need to be able to update the deployment config rather than patching the replication controller.